astro/src/content/docs/_shared/_token-storage-options.mdx

```diff
@@ -0,0 +1,15 @@
+|-----------------------|---------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|--------------------------------------|---------------------------------------|
+| `Secure`, `HTTPOnly` cookies | Tokens sent automatically when `credentials` option sent, widely supported | Won't work if clients are not on same domain as the APIs, requires a browser, APIs must look for access token in cookie header, not in other places | Reduces attack surface by preventing JavaScript from accessing tokens, XSS resistant, but requires correct cookie configuration and same-domain policies. | <GreenCheck />                                    |
+| Backend for frontend (BFF)/Sessions | Easy to revoke tokens, works with any client, can be combined with cookie approach to provide cross domain access if some APIs live on different domains                               | Additional server side component, less scalable because you are routing all API requests through the BFF, single point of failure for all API access              | Minimizes token exposure by never sharing tokens with the client, but introduces a new critical infrastructure component that must be secured | <GreenCheck />                                    |
+| Native secure storage   | OS-provided secure storage options like iOS Keychain or Android Keystore prevent token exfiltration       | Platform-specific implementation | Prevents exfiltration by malicious apps, use TLS to prevent attacker-in-the-middle attacks                    | <GreenCheck />                                    |
+| Mutual-TLS (MTLS)                 | Tokens are bound to client, IETF standard                                                      | Not widely supported, requires every client to have an X.509 certificate                                | Offers strong token-binding security, but operational complexity and certificate management introduce significant overhead                    | <GreenCheck /> (when available) |
+| Distributed Proof of Possession (DPoP)                  | Tokens are bound to client, IETF standard                                                      | Not widely supported, requires APIs to do extra work to verify the token, client must do extra work to send the token, still must take care to avoid exfiltration of key | Enhances token security by binding it to a client, but susceptible to implementation flaws and requires stringent key management              | <GreenCheck /> (when available) |
+| Local Storage, IndexedDB         | JavaScript can access token and send requests using `Authorization` or other expected header, supported by some frameworks (Amplify) | Tokens vulnerable to exfiltration by any JavaScript running on the page                                                             | Highly vulnerable to XSS attacks | <RedX /> |
```

Comment:
```suggestion
| Local Storage, IndexedDB ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)), a type of local storage)        | JavaScript can access token and send requests using `Authorization` or other expected header, supported by some frameworks (Amplify) | Tokens vulnerable to exfiltration by any JavaScript running on the page                                                             | Highly vulnerable to XSS attacks | <RedX /> |
```

I didn't know the proper name of IndexedDB, maybe link as an explainer

Reply:
Not sure if it's worth adding this as well, in-case the word local storage to too loose/ambiguous, but I think that most should know it.

 https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API. 

Dealers choice, though.